Scenarios/ITU-R_M2412-0_5BA.m

```diff
@@ -0,0 +1,50 @@
+clear all 
+
+% Disable cast to struct warnings
+w = warning ('off','all');
+
+%% Get configuration
+Config = MonsterConfig(); % Get template config parameters
+
+%add scenario specific setup for 'ITU-R M2412-0 5.B.A':
+% from https://www.itu.int/dms_pub/itu-r/opb/rep/R-REP-M.2412-2017-PDF-E.pdf Table 5.c Configuration A
+    %For Spectral efficiency and mobility Evaluations.
+    Config.Scenario = 'ITU-R M.2412-0 5.B.A';
+    Config.MacroEnb.number = 19;
+    Config.MicroEnb.number = 0;
+    Config.PicoEnb.number = 0;				
+    Config.Phy.downlinkFrequency = 4000; %MHz
+    Config.MacroEnb.height= 25;
+    Config.MacroEnb.Pmax = 10^(41/10)/1e3; %41dBm converted to W
+    Config.MacroEnb.subframes = 50; % 10MHz bandwidth
```

Comment:
Should be resource blocks right?

Reply:
This is clearly an error. I'll look into fixing this.